dspace-api/src/main/java/org/dspace/external/OpenaireRestConnector.java

```diff
@@ -99,11 +99,11 @@ public OpenAIRERestConnector(String url) {
      * 
      * @throws IOException
      */
-    public OpenAIRERestToken grabNewAccessToken() throws IOException {
+    public OpenaireRestToken grabNewAccessToken() throws IOException {
 
         if (StringUtils.isBlank(tokenServiceUrl) || StringUtils.isBlank(clientId)
                 || StringUtils.isBlank(clientSecret)) {
-            throw new IOException("Cannot grab OpenAIRE token with nulls service url, client id or secret");
+            throw new IOException("Cannot grab Openaire token with nulls service url, client id or secret");
```

Comment:
OpenAIRE with AIRE all caps (it's the project name). I understand that you need to search&replace all occurrences, but the ones that interact with the user, shouldn't be changed.

Reply:
On top of the class declaration the url to the official OpenAIRE documentation has been linked already.